docs/DeveloperGuide.md

```diff
@@ -248,6 +248,44 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
+### Help Window
+
+Syntax: ```help``` 
+<br>
+Purpose: Provides users with a bird's eye view of the various methods usable in OfficeConnect.
+
+#### Implementation
+
+A Help Window will be opened either by entering the `help` command, or by clicking on "Help" button in the toolbar. 
+The Activity Diagram below details the workflow of a user who wishes to access the HelpWindow. If the Help Window does not provide enough
+information, the user can choose to view the comprehensive User Guide instead.
+
+<img src="images/HelpActivityDiagram.png" width="330" />
```

Comment:
I think before submission we could decide on the size of images to have some sort of alignment across the guide